src/xtopcom/xblockstore/src/xvblockhub.cpp

```diff
@@ -848,6 +848,38 @@ namespace top
             return (int)it->second.size(); //found existing ones
         }
 
+        int   xblockacct_t::load_index_by_height(const uint64_t target_height)
```

Comment:
bool xblockacct_t::update_index_by_height

or

size_t xblockacct_t::update_index_by_height

usually `int` as the return value of a complex logic, it usually represents error code and 0 means successful.
some APIs in this class use `int` as the return value but not follow the guidance. we shouldn't follow it.